CleanSwift/CleanSwift/Scene/DetailNewsScene/DetailNewsWorker.swift

```diff
@@ -0,0 +1,35 @@
+// DetailNewsWorker.swift
+// Copyright © Natasha Ananas. All rights reserved.
+
+import Foundation
+
+final class DetailNewsWorker {
+    private let photoLoadService: PhotoLoadServiceProtocol
```

Comment:
```suggestion
    private let photoLoadService: PhotoLoadServiceProtocol
```
марки и энам ниже должен быть вверху